picard/ui/infodialog.py

```diff
@@ -296,6 +297,20 @@ def _display_info_tab(self):
             tabWidget.setTabText(tab_index, _("&Info"))
             self.tab_hide(tab)
 
+class TrackInfoDialog(InfoDialog):
```

Comment:
Maybe something's wrong on my end, but it doesn't seem to be possible to access this dialog from the Info button in the toolbar or the context menu of tracks without files?